boilerplates/be-docker-plain/Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+def final projectId = '@project_id@'
+def final componentId = '@component_id@'
+def final credentialsId = "${projectId}-cd-cd-user-with-password"
+def sharedLibraryRepository
+def dockerRegistry
+node {
+  sharedLibraryRepository = env.SHARED_LIBRARY_REPOSITORY
+  dockerRegistry = env.DOCKER_REGISTRY
+}
+
+library identifier: 'ods-library@0.1-latest', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: sharedLibraryRepository,
+   credentialsId: credentialsId])
+
+// See readme of shared library for usage and customization.
+odsPipeline(
+  image: "${dockerRegistry}/cd/jenkins-slave-nodejs8-angular",
```

Comment:
Maybe here we should add a comment too, that you can swap this for another ODS-provided slave or your own custom image.

Reply:
@michaelsauter what about talking the Base Slave? Do u know if i can use that for building?

Reply:
@clemensutschig I suggest to point to the base slave one but also to add the comment to load specific jenkins slave later on to fit for testing stages, if any. If there is a slave that does not fit we can later on create an issue for creating one.

Reply:
I like the idea to use the base slave, definitely works ...

Reply:
@michaelsauter @gerardcl  - tested - works :) this is cool, because it also contains proxy / sqjar etc :)